src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -401,3 +437,22 @@ const ArtworkPlaceholder: React.FC<{}> = ({}) => {
     </View>
   )
 }
+
+const BelowTheFoldPlaceholder: React.FC<{}> = ({}) => {
+  return (
+    <ProvidePlaceholderContext>
+      <Separator />
+      <Spacer mb={3} />
+      {/* About the artwork title */}
+      <PlaceholderText width={60} />
+      <Spacer mb={2} />
+      {/* About the artwork copy */}
+      <PlaceholderRaggedText numLines={4} />
+      <Spacer mb={3} />
+      <Separator />
+      <Spacer mb={3} />
+      <ActivityIndicator />
```

Comment:
Do we want to use `Spinner` from `lib/Components` here instead of the iOS indicator?

Reply:
Not sure about _we_ but _I_ don't for reasons outlined here: https://artsy.slack.com/archives/C02BGC5PW/p1580824659012300

Plus we already use the standard iOS activity indicator for pull-to-refresh on this page.

Reply:
Gotcha, just catching up on that discussion now. I agree – the spinner is super-distracting and, in this case specifically, we're trying to do something without drawing attention to the UI. So it's better to use the default.

A part of me will always love the spinner, but I agree with Jeffrey, it feels very "old Artsy" design.

![f171d1911d81f5e4](https://user-images.githubusercontent.com/498212/74356762-e28ef000-4d8c-11ea-939b-b1ca4748f5ab.gif)

RIP